Tests/MountebankSwiftTests/Models/__Imposters__/ImposterTests/testWriteToDisk.stubs.all.swift

```diff
@@ -0,0 +1,240 @@
+            predicates: [
+                .equals(Request(
+                    method: .put,
+                    path: "/test-is-200",
+                    query: ["key": [
+                        "first",
+                        "second",
+                    ]],
+                    headers: ["foo": "bar"],
+                    data: ["baz"]
+                )),
+                .deepEquals(Request(query: ["key": [
+                    "first",
+                    "second",
+                ]])),
+                .contains(Request(data: "AgM=")),
+                .startsWith(Request(data: "first")),
+                .endsWith(Request(data: "last")),
+                .matches(Request(data: "^first")),
+                .exists(Request(query: [
+                    "q": true,
+                    "search": false,
+                ])),
+                .not(.equals(Request(
+                    method: .put,
+                    path: "/test-is-200",
+                    query: ["key": [
+                        "first",
+                        "second",
+                    ]],
+                    headers: ["foo": "bar"],
+                    data: ["baz"]
+                ))),
+                .or([
+                    .equals(Request(
+                        method: .put,
+                        path: "/test-is-200",
+                        query: ["key": [
+                            "first",
+                            "second",
+                        ]],
+                        headers: ["foo": "bar"],
+                        data: ["baz"]
+                    )),
+                    .contains(Request(data: "AgM=")),
+                ]),
+                .and([
+                    .equals(Request(
+                        method: .put,
+                        path: "/test-is-200",
+                        query: ["key": [
+                            "first",
+                            "second",
+                        ]],
+                        headers: ["foo": "bar"],
+                        data: ["baz"]
+                    )),
+                    .deepEquals(Request(query: ["key": [
+                        "first",
+                        "second",
+                    ]])),
+                ]),
+                .inject("(config) => { return config.request.headers[\'Authorization\'] == \'Bearer <my-token>\'; }"),
+                .equals(
+                    Request(path: "/with-parameters"),
+                    PredicateParameters(
+                        caseSensitive: true,
+                        except: "^The ",
+                        xPath: XPath(
+                            selector: "//a:title",
+                            namespace: ["a": "http://example.com/book"]
+                        ),
+                        jsonPath: JSONPath(selector: "$..title")
+                    )
+                ),
+            ]
+        ),
+    ]
+}
+
+// swiftlint:enable line_length force_unwrapping
```

Comment:
This output file of the test is now under version control so that I can format it with `swiftformat` and see if it changes :D